src/config.ts

```diff
@@ -485,6 +494,35 @@ export class KubeConfig implements SecurityAuthentication {
         const config: Configuration = createConfiguration({
             baseServer: baseServerConfig,
             authMethods: authConfig,
+            // This HTTP API implementation is a copy of ./gen/http/isomorphic-fetch.ts,
+            // extended with the timeout pass-through code.
+            httpApi: {
+                send(request: RequestContext): Observable<ResponseContext> {
+                    const method = request.getHttpMethod().toString();
+                    const body = request.getBody();
+
+                    const resultPromise = fetch(request.getUrl(), {
+                        method: method,
+                        body: body as any,
```

Comment:
```suggestion
                        body: body as BodyInit,
```

If we do this typecast we should at least cast into the correct type.

Reply:
Could do that. Would lose the pure symmetry with the original code then. If you're aware that I deliberately copied the `send` implementation verbatim, trying to stay close to the original code, but still prefer to have the more correct type cast, I can do that!